### ISSUE TITLE
INCLUDE TYPE AS, false positive, 2

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1917]
```abap
TYPES: BEGIN OF bar,
         field TYPE c LENGTH 1,
       END OF bar.
TYPES: BEGIN OF something,
         field1 TYPE c LENGTH 1.
         INCLUDE TYPE bar AS mo.
TYPES END OF something.
DATA moo TYPE something.
WRITE moo-field.
```
